pages/contact-us.md

```diff
@@ -66,6 +66,6 @@ Contributions to Apache Accumulo are welcome! If you are interested, read [how t
 [N_UN]: mailto:notifications-unsubscribe@accumulo.apache.org
 {: .btn .btn-danger .btn-xs }
 
-[accumulo-irc]: irc://chat.freenode.net/accumulo
-[freenode]: https://freenode.net/
+[asf-slack]: https://the-asf.slack.com/
```

Comment:
I think may be a https://the-asf.slack.com/messages/CERNB8NDC is a permalink to the channel

Reply:
Added a link to channel